modules/stitching/src/camera.cpp

```diff
@@ -63,10 +63,12 @@ const CameraParams& CameraParams::operator =(const CameraParams &other)
 
 Mat CameraParams::K() const
 {
-    Mat_<double> k = Mat::eye(3, 3, CV_64F);
-    k(0,0) = focal; k(0,2) = ppx;
-    k(1,1) = focal * aspect; k(1,2) = ppy;
-    return k;
+  Mat_<double> k = Mat::eye(3,3, CV_64F);
+  k(0,0) = focal;
+  k(0,2) = ppx;
+  k(1,1) = focal  * aspect;
+  k(1,2) = ppy;
+  return k;
```

Comment:
For new source branch it is needed to open new PR. GitHub allows to change target branch only at this moment.